src/api/exporters/ezsLocals/JSONLDObject.js

```diff
@@ -61,15 +61,28 @@ function mergeSimpleField(output, field, data) {
     };
 }
 
-function getUri(uri) {
-    if (uri.indexOf('http://') !== 0 &&
-        uri.indexOf('https://') !== 0) {
-        return path.normalize(this.getParam('hostname', 'http://lod.istex.fr/').concat(uri));
+function removeNumberInstance(uri) {
+    const reg = new RegExp('(\\-\\d+)(\\.[a-z]+)+');
+    const match = reg.exec(uri);
+
+    if (match !== null) {
+        return uri.replace(match[1], '');
     }
 
     return uri;
 }
 
+function getUri(uri) {
+    const u = removeNumberInstance(uri);
+
+    if (u.indexOf('http://') !== 0 &&
+        u.indexOf('https://') !== 0) {
+        return path.normalize(this.getParam('hostname', 'http://lod.istex.fr/').concat(u));
```

Comment:
@touv Why do you want to append `http://lod.istex.fr/` to any URI?
It should be what is called `host` (`http://wos-category.lod.istex.fr/`) in [default.js](https://github.com/Inist-CNRS/lodex/blob/master/config/default.js#L10), shouldn't it?

Besides, in what cases `u` does not begin with `http`?
default.js set a default value to `host`, beginning with `http`.